man/man4/zfs.4

```diff
@@ -726,10 +730,10 @@ ARC target size
 .Pq Sy arc_c
 by thresholds determined by this parameter.
 Exceeding by
-.Sy ( arc_c >> zfs_arc_overflow_shift ) * 0.5
+.Sy ( arc_c No >> Sy zfs_arc_overflow_shift ) No / Sy 2
 starts ARC reclamation process.
 If that appears insufficient, exceeding by
-.Sy ( arc_c >> zfs_arc_overflow_shift ) * 1.5
+.Sy ( arc_c No >> Sy zfs_arc_overflow_shift ) No \(md Sy 1.5
```

Comment:
```suggestion
.Sy ( arc_c No >> Sy zfs_arc_overflow_shift ) No \(mu Sy 1.5
```

I'm not an mandoc expert, but according to the [documentation](https://mandoc.bsd.lv/man/mandoc_char.7.html) you want `\(mu` here (and throughout).  This does render correctly for me.

Reply:
Interesting, mandoc does seem to be missing this (or, indeed, calls it `\(pc`, which groff also has?), while groff has it since its inception. Didn't expect *that* to be the GNU mdoc extension that bit me in the ass today! Fixed as suggested.